### DISCUSSION TITLE
Fix Error 405 when following creating-a-new-operation-with-no-interface article.

### DISCUSSION THREAD

**Discussion by christmex** [Laravel-Backpack/community-forum#485] [General] [answered]
Hi @tabacitu, so today I'm making a new custom operation, and then I use this article as my reference,
https://backpackforlaravel.com/docs/5.x/crud-operations#creating-a-new-operation-with-no-interface

but then, there is a little mistake in that article I think. I hope I'm not wrong with this one 😓, please take a look at this step

in this step, we're using the GET method
![image](https://github.com/Laravel-Backpack/community-forum/assets/34405837/7602af2a-91df-4837-a1d2-73ffdcbbd1d2)

but in our custom button, we use POST method
![image](https://github.com/Laravel-Backpack/community-forum/assets/34405837/f28c3911-b15b-49a4-84f2-229f83862539)

so I think we need to change the GET method into POST method in our controller, right? cuz I'm trying to still use the GET method in our controller and ended up with this error 
![image](https://github.com/Laravel-Backpack/community-forum/assets/34405837/153dc589-d2a9-45ac-b336-96efac9f9fba)

Thanks,...

**Comment by jcastroa87** [comment, marked as answer]
Hi @christmex thanks a lot about this.

I will fix it ASAP.

Cheers.
